contact.go

```diff
@@ -361,3 +361,11 @@ func (s *ContactServiceOp) AssociateAnotherObj(contactID string, conf *Associati
 	}
 	return resource, nil
 }
+
+// Delete deletes a contact.
+func (s *ContactServiceOp) Delete(contactID string) error {
```

Comment:
nits: It can also be written as a one-liner.
```go
func (s *ContactServiceOp) Delete(contactID string) error {
	return s.client.Delete(s.contactPath + "/" + contactID)
}
```

Reply:
Could you please move it between `Update` and `AssociateAnotherObj`? 
Simple CRUD operations would like to be listed nearby.